e2e/addObservationWithoutData.e2e.js

```diff
@@ -0,0 +1,48 @@
+import {
+  by,
+  device,
+  element,
+  expect,
+  waitFor
+} from "detox";
+
+describe( "Add observation without evidence", () => {
+  beforeAll( async () => {
+    await device.launchApp( {
+      newInstance: true,
+      permissions: { location: "always" }
+    } );
+  } );
+
+  beforeEach( async () => {
+    // device.launchApp is preferred for an app of our complexity. It does work locally for both,
+    // but on CI for Android it does not work. So we use reloadReactNative for Android.
+    if ( device.getPlatform( ) === "android" ) {
+      await device.reloadReactNative( );
+    } else {
+      await device.launchApp( {
+        newInstance: true,
+        permissions: { location: "always" }
+      } );
```

Comment:
Aren't we already doing this in the `beforeAll` above? Or do we need to do it twice?

Reply:
In order to get passing tests on CI on Android, for some reason I had to change to use `device.reloadReactNative( )` even though the preferred `device.launchApp` was working fine on my local machine. So, for Android we need to launch the App in the beforeAll call. On iOS it is redundant with the first time it hits beforeEach, correct. But it does not take a lot of time, and I wanted to avoid another check for Platform.